front/src/components/post-button-and-function/postButton.component.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { postFunction } from "./postFunction";
+
+export const postButton = (startDate, endDate) => {
```

Comment:
please use PascalCase to name components (and their files)

Reply:
fixed